src/main/target/AG3X/target.h

```diff
@@ -170,5 +170,5 @@
 #define TARGET_IO_PORTC (0xffff)
 #define TARGET_IO_PORTD BIT(2)
 
-#define USABLE_TIMER_CHANNEL_COUNT 15
-#define USED_TIMERS ( TIM_N(1) | TIM_N(2) | TIM_N(3) | TIM_N(4) | TIM_N(5) | TIM_N(8) | TIM_N(9) | TIM_N(12))
+#define USABLE_TIMER_CHANNEL_COUNT ((15 + BBSHOT_PACER_COUNT) + BBSHOT_PACER_COUNT)
```

Comment:
Same as above, should be handled in the code consuming `USABLE_TIMER_CHANNEL_COUNT`.